packages/client/src/generation/defaults/defaultBrowserIndex.ts

```diff
@@ -0,0 +1,13 @@
+export const defaultBrowserIndex = `class PrismaClient {
```

Comment:
These aren't files that we copy anymore, but bundle-able contents (used by the generator and postinstall).